scripts/install-build-deps

```diff
@@ -115,10 +118,7 @@ pip3 install ipaddress
 
 if ! which ansible &>/dev/null ; then
     case $(distro_type) in
-        redhat) yum -y install epel-release
-                yum -y install epel-release # Update it
-                yum -y install ansible
-                ;;
+        redhat) yum -y install ansible ;;
```

Comment:
Why? The old code made sure that the latest epel-release is installed and the latest ansible is used.